run_linux.go

```diff
@@ -198,9 +198,19 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	}
 	rootIDPair := &idtools.IDPair{UID: int(rootUID), GID: int(rootGID)}
```

Comment:
This should move down 20 or 30 lines towards line 225 where it's first referenced.